tests/vad/test_webrtc.py

```diff
@@ -0,0 +1,36 @@
+"""
+This module contains tests for Voice Activity Detection
+"""
+from unittest.mock import patch
+
+import numpy as np
+
+from spokestack.context import SpeechContext
+from spokestack.vad.webrtc import VoiceActivityDetector
+
+
+@patch("spokestack.vad.webrtc.webrtcvad")
+def test_vad(mock_class):
+    context = SpeechContext()
+    detector = VoiceActivityDetector(
+        sample_rate=16000,
+        frame_width=10,
+        vad_rise_delay=0,
+        vad_fall_delay=20,
+        mode="quality",
+    )
+
+    # test detection
+    frame = np.zeros(160, np.int16).tobytes()
+    detector(context, frame)
```

Comment:
you're testing two calls here to trigger the vad, but you have vad_rise_delay set to 0 above, so it should actually becoming triggered after the first call